### PR TITLE
V10 update

### PR DESCRIPTION
**Pull request by eXaminator**
Hi there,

I hope you don't mind the PR. This module is the last one missing for me to go to V10, so I thought I'd just see if I can update it myself and either have a working version for me or maybe save you some time.

All the updates and data management is pretty complex so I'm bound to have missed something somewhere, but it seems to work for me so far as far as I can tell. I also fixed a deprecation message from the PF1 system that was introduced with 0.82.0.

Please take this with some grain of salt and give it a good look if you consider merging this! :)

### CODE REVIEW

File: module.json [hunk 1]
```diff
@@ -1,21 +1,31 @@
...
             "discord": "websterguy#1136"
         }
     ],
-    "system": [
-        "pf1"
-    ],
+    "systems": [{
+        "id": "pf1",
+        "type": "system",
+        "manifest": "https://gitlab.com/foundryvtt_pathfinder1e/foundryvtt-pathfinder1/-/releases/v0.82.2/downloads/system.json",
```
Comment (mkahvi) [inline module.json:24]: The manifest link for the system is unnecessary detail.

File: module.json [hunk 2]
```diff
@@ -1,21 +1,31 @@
...
             "url": "https://github.com/websterguy",
             "discord": "websterguy#1136"
         }
     ],
-    "system": [
-        "pf1"
-    ],
+    "systems": [{
+        "id": "pf1",
+        "type": "system",
```
Comment (mkahvi) [inline module.json:23]: `systems` don't need type. They're all implicitly systems.

File: module.json [hunk 3]
```diff
@@ -1,21 +1,31 @@
...
-    "name": "pf1-mighty-morphin",
+    "id": "pf1-mighty-morphin",
     "title": "PF1 Might Morphin Mod",
     "description": "Mod to automate polymorph effects for Foundry VTT PF1e system",
     "version": "0.3.0",
     "manifestPlusVersion": "1.0.0",
-    "minimumCoreVersion": "9",
-    "compatibleCoreVersion": "9",
+    "minimumCoreVersion": 10,
+    "compatibleCoreVersion": 10,
```
Comment (mkahvi) [inline module.json:8]: These are obsolete since other v9 fields have been removed.

Comment (eXaminator) [reply module.json:8]: I know. But v9 can't read the new syntax and thus will try to install the new module version, breaking everything for users (had that with my own module at first). So I'd suggest to leave it in and removeitsometime down the line when v9 has mostly phased out.

**Comment by mkahvi** [comment]
Also, `.img` is incorrect for v10, it's `.texture.src` in the TokenDocument (not in Token), so requires checking if you need `token.document.texture.src` or `token.texture.src`

**Comment by eXaminator** [comment]
> Also, `token.img` is incorrect for v10, it's `.texture.src` in the TokenDocument (not in Token), so requires checking if you need `token.document.texture.src` or `token.texture.src`.
> 
> For prototype token it's `actor.prototypeToken.texture.src` (instead of `actor.token.img`)

It should be fixed. Sadly my dev installation is kind of borked right now and I never set it up to actually change token images in the first place. So I can't test this propperly. I hope someone else will be able to see if I got everything right by giving it a spin.

**Pull request closed without merging**